### PR TITLE
Add Japanese

### PR DESCRIPTION
**Pull request by maiwallet**

### CODE REVIEW

File: Support-xApp-Monkdesk/ja.json [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+, "session_expired": "セッションの有効期限が切れています。"
+, "open_again": "セキュリティ上、XUMMサポートxAppのセッションタイムは短く設定されています。お手数ですがもう一度アプリを開いてください。"
+, "reason:": "理由:"
+, "only_in_app": "XUMMサポートはアプリ内からのみ利用可能です。"
+, "app_or_support_center": "アプリ内のXUMMサポートxAppから、または{{ support_center }}からお問い合わせください。"
+, "support_center": "Xummサポートセンター"
+, "no_read_only": " <q><b>読み取り専用アカウント</b></q>からはお問い合わせいただくことができません。"
+, "switch_full_access": "ホーム画面の右上<q><b>アカウント</b></q>をタップして、全権アクセスのあるアカウントからお問い合わせください。"
+, "no_full_access_mail": "<q>全権アクセス</q>があるアカウントがない場合は、{{ mailaddr }}までメールでお問い合わせをお願いします"
+, "warning_version": "<b>注意！</b> お使いのバージョン<b>{{ current_version }}</b>以降、新しいバージョン<b>{{ latest_version }}</b>がリリースされています。以前のバージョンに起因する問題を排除するため、お手数ですが<b>最新のバージョンにアップデート</b>をお願いします。"
```
Comment (develoQ) [inline Support-xApp-Monkdesk/ja.json:21]:
```suggestion
, "warning_version": "<b>注意！</b> 現在お使いのバージョンは<b>{{ current_version }}</b>ですが、新しいバージョン<b>{{ latest_version }}</b>がリリースされています。以前のバージョンに起因する問題を排除するため、お手数ですが<b>最新のバージョンにアップデート</b>をお願いします。"
```

File: Support-xApp-Monkdesk/ja.json [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+, "ticket_not_found": "お問い合わせ内容が見つかりません"
+, "trouble_finding_ticket": "恐れ入りますが、お問い合わせいただいた内容を表示できません。ご迷惑をおかけし申し訳ございません。🙏"
+, "new_ticket": "新しいお問合せ"
```
Comment (develoQ) [inline Support-xApp-Monkdesk/ja.json:24]:
```suggestion
, "new_ticket": "新しいお問い合せ"
```

File: Support-xApp-Monkdesk/ja.json [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+, "ask_q": "お問い合わせ内容"
+, "maintenance": "現在メンテナンス中です。しばらく時間をおいてから再度お試しください。"
+, "how_can_we_help": "いかがいたしましたか？"
+, "agents_will_help": "準備ができ次第{{ agent_names }}がお答えします"
+, "ticket_state_closed": "解決済み"
+, "ticket_state_pending": "進行中"
+, "ticket_state_awaiting_your_reply": "返信待ち"
+, "ticket_no": "お問合せ番号"
```
Comment (develoQ) [inline Support-xApp-Monkdesk/ja.json:32]:
```suggestion
, "ticket_no": "お問い合せ番号"
```

**Review by develoQ — APPROVED** [review]
It’s OK.
I suggest clearer wording and consistent wording.

**Comment by WietseWind** [comment]
Thank you, both 🤗

**Comment by WietseWind** [comment]
IMG_3768

**Pull request merged**